datumaro/plugins/synthetic_images_plugin/image_generator.py

```diff
@@ -0,0 +1,174 @@
+# Copyright (C) 2022 Intel Corporation
+#
+# SPDX-License-Identifier: MIT
+
+from multiprocessing import Pool
+from shutil import rmtree
+from typing import List, Optional
+import os
+
+import cv2 as cv
+import numpy as np
+
+from datumaro.components.cli_plugin import CliPlugin
+from datumaro.components.errors import SourceExistsError
+
+from .utils import IFSFunction, augment, colorize, download_colorization_model
+
+
+class ImageGenerator(CliPlugin):
```

Comment:
I don't think inheritance from `CliPlugin` (as well as the implementation of `build_cmdline_parser`) does anything here. `CliPlugin` is only useful as a base class in other plugin types (which are listed in the `plugin_types` function).

Reply:
We discussed with @zhiltsov-max, that I should add a new plugin type **(inherit from CliPlugin)**, which allows to generate datasets

Reply:
I added `--type` instead of `-g` because we don't have specific algo name

```python
    parser.add_argument('-t', '--type', required=True, choices=['image'],
        help="Specify type of data to generate")
```